rasa/core/test.py

```diff
@@ -1005,7 +981,6 @@ async def test(
         out_directory: path to directory to results to
         fail_on_prediction_errors: boolean indicating whether to fail on prediction
             errors or not
-        e2e: boolean indicating whether to use end to end evaluation or not
```

Comment:
The `e2e` argument is still present in the function signature, shouldn't it be removed too?

Reply:
It's still needed here — e2e is still used by `_collect_story_predictions` and other functions within test module.

Reply:
Oh I see, I got confused by the removal in the docstring.

Reply:
That's a mistake — I will return it back.